src/ducks/knownAccounts.ts

```diff
@@ -0,0 +1,52 @@
+import { createAsyncThunk, createSlice } from "@reduxjs/toolkit";
+import {
+  ActionStatus,
+  KnownAccount,
+  KnownAccountsInitialState,
+} from "types/types.d";
+
+export const fetchAccountsAction = createAsyncThunk<KnownAccount[]>(
+  "knownAccounts/fetchKnownAccountsAction",
+  async () => {
+    let result;
+    try {
+      const response = await fetch(
+        "https://api.stellar.expert/explorer/directory?tag[]=memo-required",
```

Comment:
I'm  wondering if the name "known" account makes sense for what this endpoint returns.

Maybe to make more specific, `MemoRequiredAccount`?

Reply:
That's a good point. I'll update it to make it more clear.